[2024-12-04T14:34:38.206+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: FoReport_file_processing.etl_merge_FO manual__2024-12-04T14:27:07.572431+00:00 [queued]>
[2024-12-04T14:34:38.518+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: FoReport_file_processing.etl_merge_FO manual__2024-12-04T14:27:07.572431+00:00 [queued]>
[2024-12-04T14:34:38.528+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 2
[2024-12-04T14:34:38.875+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): etl_merge_FO> on 2024-12-04 14:27:07.572431+00:00
[2024-12-04T14:34:38.957+0000] {standard_task_runner.py:57} INFO - Started process 374 to run task
[2024-12-04T14:34:39.005+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'FoReport_file_processing', 'etl_merge_FO', 'manual__2024-12-04T14:27:07.572431+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/FoReport_file_processing.py', '--cfg-path', '/tmp/tmp56pei41b']
[2024-12-04T14:34:39.013+0000] {standard_task_runner.py:85} INFO - Job 63: Subtask etl_merge_FO
[2024-12-04T14:34:39.545+0000] {task_command.py:410} INFO - Running <TaskInstance: FoReport_file_processing.etl_merge_FO manual__2024-12-04T14:27:07.572431+00:00 [running]> on host effe9442f8f2
[2024-12-04T14:34:39.805+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='FoReport_file_processing' AIRFLOW_CTX_TASK_ID='etl_merge_FO' AIRFLOW_CTX_EXECUTION_DATE='2024-12-04T14:27:07.572431+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-04T14:27:07.572431+00:00'
[2024-12-04T14:34:50.324+0000] {logging_mixin.py:150} INFO - No data to merge.
[2024-12-04T14:34:50.397+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-12-04T14:34:50.632+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=FoReport_file_processing, task_id=etl_merge_FO, execution_date=20241204T142707, start_date=20241204T143438, end_date=20241204T143450
[2024-12-04T14:34:50.780+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-12-04T14:34:50.860+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
